### DISCUSSION TITLE
Custom padding in Forms Plugin

### DISCUSSION THREAD

**Discussion by ishaiavrahami** [tailwindlabs/tailwindcss#8929] [Help]
Hello, I'm trying to insert custom padding to inputs, selects.. in the forms plugin. Currently using Vue and latest forms plugin. Any suggestions?

<img width="313" alt="Screen Shot 2022-07-22 at 1 51 19 PM" src="https://user-images.githubusercontent.com/33962719/180424435-1d89c5f9-3e1f-4b81-9efb-d501322d3280.png">

**Comment by wongjn** [comment]
You should be able to use the padding classes on any of the form elements, just like any other HTML element.
